test/blocks/chart/chart.test.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-expressions */
+/* global describe it */
+
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+
+const { getChartOptions } = await import('../../../libs/blocks/chart/chart.js');
+
+describe('chart', () => {
+  it('get bar chart options', async () => {
+    document.body.innerHTML = await readFile({ path: './mocks/bar.html' });
+    const options = getChartOptions('bar', {}, []);
+    expect(options).to.exist;
```

Comment:
Is the only way this tests fails is if options === undefined || false || null ? I believe, getChartOptions is always going to return an object, regardless of the params. 

There are some internal functionalities going on in getChartOptions, like deleting chartData, modifying and shifting seriesData. Would there be better ways to test these with dummy data, or would it require too much setup?

Reply:
That sounds fine 👍